pipeline/default.go

```diff
@@ -0,0 +1,15 @@
+package pipeline
+
+import (
+	"fmt"
+
+	"github.com/goreleaser/goreleaser/context"
+)
+
+// Defaulter interface
```

Comment:
Maybe can be a bit more descriptive here, something like:
`Defaulter can be implemented by a Piper to set default values for its configuration.`